undebug/main.py

```diff
@@ -2,8 +2,9 @@
 import subprocess
 
 def run_command(command, pager=False):
-    click.echo(click.style("Command: ", fg='cyan') + click.style(command, fg='green'))
-    p = subprocess.Popen(command, shell=True, text=True, stdout=subprocess.PIPE)
+    command_str = ' '.join(command)
```

Comment:
>join [](http://example.com/codeflow?start=22&length=4)

Is it better to use `shlex.join`? This will be applicable to multiple places.

Reply:
I have a concern, `shlex.join` is available starting Python 3.8, I don't think we can use this